### DISCUSSION TITLE
Google custom search overlay is partially hidden

### DISCUSSION THREAD

**Discussion by isohedral** [twbs/bootstrap#35620] [Q&A]
Hi, I find myself surprisingly stuck on a Bootstrap issue (normally the docs walk me through everything!), so I thought I'd come here.

I recently upgraded the website archive.bridgesmathart.org to Bootstrap 5.1.3. Every page's sidebar contains a Google Custom Search field, using their latest embed code. On desktop Google Chrome, the search results (in overlay mode) appear correctly on top of everything else on the page.  But on mobile browsers, Safari, and I think Firefox, the search results cover just the sidebar, with the rest hidden under the main part of the page. Bootstrap's z-indices are all in the 1000 neighbourhood, and Google's are all around 100000, so I can't understand how anything is getting hidden.  Any suggestions would be most welcome, thanks.

(This is basically a re-post of a question on Google's Programmable Search Engine message board, where I have not yet received a useful reply.)

**Comment by Albert7866** [comment]
The issue with the Google Custom Search overlay seems related to z-index stacking. While z-index values are set appropriately, differences in browser behavior, especially on mobile browsers like Safari and potentially Firefox, might be affecting how elements stack.

Using browser developer tools to inspect the [elements](https://gloryshinestore.com/collection/hayabusa-suits/) and their stacking contexts can help identify any browser-specific CSS or default styles causing this issue. Experimenting with z-index values and reviewing any browser-specific quirks might resolve the problem. Ensure consistent z-index application across sidebar and main content elements for uniform overlay behavior.
